src/common/msgout.cpp

```diff
@@ -150,7 +150,7 @@ void wxMessageOutputStderr::Output(const wxString& str)
     const wxString strWithLF = AppendLineFeedIfNeeded(str);
     const wxWX2MBbuf buf = strWithLF.mb_str();
 
-    if ( buf )
+    if ( buf && strlen(buf) > 0 )
```

Comment:
It's a micro optimization but it's better to test for `buf[0] != '\0'` to avoid computing the length of a possibly long string needlessly.

And if we wanted to support strings with embedded NULs properly, we should use `buf.length()` instead, in Unicode build only.

Maybe the best would be to provide some kind of `wxIsBufferEmpty()` function.